File: Dataset/Leetcode/test/35/73.py
 class Solution(object):
    def XXX(self, nums, target):
        j=len(nums)-1
        for i in range(j):
            if target==nums[i]:
                return i
            else:
                if target>nums[i] and target<nums[i+1]:
                    return i+1
                else:
                    if target<nums[0]:
                        return 0
                    else:
                        return j
                    

File: Dataset/Leetcode/train/26/73.java
 class Solution {
    public int XXX(int[] nums) {
        int k=0;
        for(int i=1;i<nums.length;i++)
        {
            if(nums[i]==nums[i-1])
                k++;
            else
                nums[i-k]=nums[i];
        }
        return nums.length-k;
    }
}

